powerspace-model/src/main/protobuf/powerspace.proto

```diff
@@ -51,7 +51,18 @@ message BidExt {
     optional double ctr = 1000;
     optional double cpc = 1001;
     optional string algorithm = 1002;
-    optional double margin = 1003;
+
+    optional double advertiserCpm = 1006;
+    optional double advertiserCpc = 1004;
+
+    optional double finalRevenue = 1003; // == margin
```

Comment:
don't really like final revenue naming if it's == margin

Reply:
it's powerspace model, so maybe we can name it powerspaceRevenue or something

Reply:
Will move it to private extensions

Reply:
it doesn't change what I say ^^